cloudhub-2/modules/ROOT/pages/ps-config-fw-rules.adoc

```diff
@@ -82,3 +84,176 @@ image::fw-rule-delete.png[Delete icon on the Firewall rules tab]
 
 . Click *Save Changes* or *Discard changes*.
 . Restart your application.
+
+== Configure TCP Ports for Apps
+
+To configure protocols other than HTTP or HTTPS, configure TCP inbound rules to manage traffic for your private space. You can then use the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/amc-application-manager/minor/4.0/console/method/%231158/[Application Manager API] to deploy an app using TCP. 
+
+=== Find Free Ports for Your App
+
+The port range for TCP is from 30500 to 32500. To confirm port availability, use the (name of) API that returns a list of available ports. 
+
+The following examples show the API request and response when you use it to find free ports:
+
+Request:
+
+[source,json]
+---- 
+GET
+https:/anypoint.mulesoft.com/runtimefabric/api/organizations/{organizationId}/privatespaces/{privatespaceId}/ports?available=true&count=10
+----
+
+Response: 
+
+[source,json]
+----
+{
+   "ports": [
+       30500,
+       30501,
+       30502,
+       30503,
+       30504,
+       30511,
+       30512,
+       30513,
+       30514,
+       30515
+   ]
+}
+----
+
+
+The `available` parameter enables you to alternate between used and unused ports.
+The `count` parameter enables you to specify the amount of ports desired. `count` can range from 1 to 10. 
```

Comment:
What happens if the request is 10 ports but only 1 is available. Will it still return 1?